src/parse-mockdb.js

```diff
@@ -285,91 +422,123 @@ var MockRESTController = {
  *   ]
  * }
  */
-function handleBatchRequest(method, path, data) {
+function handleBatchRequest(unused1, unused2, data) {
   const requests = data.requests;
   const getResults = requests.map(request => {
-    var method = request.method;
-    var path = request.path;
-    var body = request.body;
-    return handleRequest(method, path, body).then(result => {
-      return Parse.Promise.as({ success: result.response });
-    })
-  })
+    const method = request.method;
+    const path = request.path;
+    const body = request.body;
+    return handleRequest(method, path, body)
+      .then(result => Parse.Promise.as({ success: result.response }));
+  });
 
-  return Parse.Promise.when.apply(null, getResults).then(function(results) {
-    return respond(200, arguments);
-  })
+  return Parse.Promise.when.apply(null, getResults).then((...args) => respond(200, args));
```

Comment:
this is what is breaking the build.  ...args is es6.  not es5.  i need to figure out what is going on.....